### PR TITLE
[solidjs-tailwinf] Add user profile card

### PR DESCRIPTION
**Pull request by vyktoremario**
Closes: #688

### CODE REVIEW

File: solidjs-tailwind/src/components/UserProfile/OrgList.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { For } from 'solid-js';
+function OrgList(props) {
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/UserProfile/OrgList.jsx:2]: Use splitProps as it shows a major feature in SolidJs

File: solidjs-tailwind/src/components/Icons/TwitterIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function TwitterIcon(className) {
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/Icons/TwitterIcon.jsx:1]: use splitProps as it shows a major feature in Solidjs

File: solidjs-tailwind/src/components/UserProfile/UserProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  star,
+  buildingOffice,
+  mapPin,
+  link,
+} from 'solid-heroicons/outline';
+import { Icon } from 'solid-heroicons';
+import { TwitterIcon } from '../Icons';
+import OrgList from './OrgList';
+
+const UserProfile = (userProfileProps) => {
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/UserProfile/UserProfile.jsx:12]: use spiltProps or merge props

File: solidjs-tailwind/src/components/UserProfile/UserProfile.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should show the user display name', async () => {
+    const fullName = await wrapper.getByText(userProfileProps.name);
+    expect(fullName).toBeVisible();
+  });
+
+  it('should have a link for user profile picture', async () => {
+    const avatar = await wrapper.getByAltText('Avatar');
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/UserProfile/UserProfile.spec.jsx:27]: does everyone have profile picture? I don't think so

File: solidjs-tailwind/src/components/UserProfile/UserProfile.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  return (
+    <div>
+      <img
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/UserProfile/UserProfile.jsx:15]: this image doesn't seem responsive. Please confirm

**Review by hdJerry — CHANGES_REQUESTED** [review]

File: solidjs-tailwind/src/components/UserProfile/OrgList.jsx (hunk 1 above)
Comment (vyktoremario) [reply solidjs-tailwind/src/components/UserProfile/OrgList.jsx:2]: @hdJerry I don't understand this comment. Can you explain why `splitProps` is required and how what you mean by ` it shows a major feature in SolidJs`?

File: solidjs-tailwind/src/components/UserProfile/UserProfile.spec.jsx (hunk 4 above)
Comment (vyktoremario) [reply solidjs-tailwind/src/components/UserProfile/UserProfile.spec.jsx:27]: GitHub gives them a placeholder if they don't upload a profile picture. That placeholder is returned as the profile picture when we query their data.

File: solidjs-tailwind/src/components/UserProfile/OrgList.jsx (hunk 1 above)
Comment (vyktoremario) [reply solidjs-tailwind/src/components/UserProfile/OrgList.jsx:2]: @hdJerry `splitProps` is used when you want to pick out certain properties from an object and pass the rest into a child component. It works the same with `Pick` in typescript.

It doesn't make sense to use it when all the properties passed are going to be used in that component.

If we need to show how major a feature it is, we can have a store that returns both user and repositories data in one object, then split it into two and pass it into their respective components(or into components where the data is needed). Or do the split inside the component and retrieve just the properties needed by the component.

File: solidjs-tailwind/src/components/UserProfile/UserProfile.jsx (hunk 5 above)
Comment (vyktoremario) [reply solidjs-tailwind/src/components/UserProfile/UserProfile.jsx:15]: The image is responsive

File: solidjs-tailwind/src/components/UserProfile/OrgList.jsx (hunk 1 above)
Comment (hdJerry) [reply solidjs-tailwind/src/components/UserProfile/OrgList.jsx:2]: If the props are not reactive then no need. @vyktoremario

**Review by hdJerry — DISMISSED** [review]

**Review by hdJerry — APPROVED** [review]

**Pull request merged**